Online-Messaging-App/src/app/register/register.component.html

```diff
@@ -1,29 +1,57 @@
 <div class="container">
-
+  <mat-toolbar color="primary" class ="navbar">
+    <div>Streamline Athletes</div>
+    <div>
+      <span>Home</span>
+      <span>About</span>
+      <span>Services</span>
+    </div>
+  </mat-toolbar>
+  <div align="center" class="mat-display-2">Streamline Athletes</div>
   <div class="row">
     <div class="col-lg-4">
       <form (ngSubmit)="registerSubmit(regform)" #regform="ngForm">
-        <div class="form-group">
-          <label for="username">Username</label>
-          <input type="text" name="username" id="username" ngModel class="form-control">
+        <div align = "center" class="form-group">
+          <mat-form-field appearance="fill" color = "primary">
+            <mat-label>Username</mat-label>
+            <input matInput type="text" name="username" id="username" ngModel class="form-control">
+          </mat-form-field>
+          <mat-form-field appearance="fill" color = "primary">
```

Comment:
Flow is wrong, when I press tab I go from username to firstname, then to password, then to lastname, then to confirm password. If you want them in two colums, please use css or material to do so, but ensure HTML flow is correct

Reply:
Done